### ISSUE TITLE
Adding CKEditor

### ISSUE THREAD

**Issue by SRocevas** [artf/grapesjs-mjml#55]
Is is posible to add [CKEditor](https://github.com/artf/grapesjs-plugin-ckeditor) in to grapesjs-mjml?

**Comment by kickbk** [comment]
Yes of course, it's really clearly explained in the docs and very straightforward. Please read the instructions.
Load the ckeditor js. You can do it via cdn too (cdnjs.cloudflare.com/ajax/libs/ckeditor/4.8.0/ckeditor.js)
Then initialize:
`plugins: ['gjs-plugin-ckeditor']`
and add your settings as described to activate:
```
pluginsOpts: {
			'gjs-mjml': {
			},
			'gjs-plugin-ckeditor': {
				...
			},
...
```

**Comment by DRoet** [comment]
seems to be answered already, thanks @kickbk
